hack/ci/testdata/konnectivity_oauth_values.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 The Kubermatic Kubernetes Platform contributors.
```

Comment:
How is this file different to [ouath_values.yaml](https://github.com/kubermatic/kubermatic/blob/69d12dbb03d6524f150be438f9c9030669d0f835/hack/ci/testdata/oauth_values.yaml) and why do we need a separate one?

Reply:
The other file is in use by other tests. 

This has different values for fields `scheme`, `host` in `dex.ingress`.

Reply:
ok :)